kubectl-minio/cmd/tenant-status.go

```diff
@@ -0,0 +1,161 @@
+	f.BoolVarP(&c.jsonOutput, "json", "j", false, "json output")
+	return cmd
+}
+
+func (d *statusCmd) validate(args []string) error {
+	return validateTenantArgs("status", args)
+}
+
+func validateTenantArgs(cmd string, args []string) error {
+	if args == nil {
+		return errors.New(fmt.Sprintf("provide the name of the tenant, e.g. 'kubectl minio tenant %s tenant1'", cmd))
+	}
+	if len(args) != 1 {
+		return errors.New(fmt.Sprintf("%s command supports a single argument, e.g. 'kubectl minio %s tenant1'", cmd, cmd))
+	}
+	if args[0] == "" {
+		return errors.New(fmt.Sprintf("provide the name of the tenant, e.g. 'kubectl minio tenant %s tenant1'", cmd))
+	}
+	return helpers.CheckValidTenantName(args[0])
+
+}
+
+func (d *statusCmd) run(args []string) error {
+	oclient, err := helpers.GetKubeOperatorClient()
+	if err != nil {
+		return err
+	}
+	if d.ns == "" || d.ns == helpers.DefaultNamespace {
+		d.ns, err = getTenantNamespace(oclient, args[0])
+		if err != nil {
+			return err
+		}
+	}
+	tenant, err := oclient.MinioV2().Tenants(d.ns).Get(context.Background(), args[0], metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	return d.printTenantStatus(tenant)
+}
+
+func (d *statusCmd) printTenantStatus(tenant *v2.Tenant) error {
+	if !d.jsonOutput && !d.yamlOutput {
+		d.printRawTenantStatus(tenant)
+		return nil
+	}
+	if d.jsonOutput && d.yamlOutput {
+		return fmt.Errorf("Only one output can be used to display status")
+	}
+	if d.jsonOutput {
+		return d.printJSONTenantStatus(tenant)
+	}
+	if d.yamlOutput {
+		return d.printYAMLTenantStatus(tenant)
+	}
+	return nil
+}
+
+func (d *statusCmd) printRawTenantStatus(tenant *v2.Tenant) {
+	gb := float32(humanize.GiByte)
+	fmt.Printf(Blue("Current status: %s \n", tenant.Status.CurrentState))
+	fmt.Printf(Blue("Available replicas: %d \n", tenant.Status.AvailableReplicas))
+	fmt.Printf(Blue("Pools: %d \n", len(tenant.Status.Pools)))
+	fmt.Printf(Blue("Revision: %d \n", tenant.Status.Revision))
+	fmt.Printf(Blue("Sync version: %s \n", tenant.Status.SyncVersion))
+	fmt.Printf(Blue("Provisioned users: %t \n", tenant.Status.ProvisionedUsers))
+	fmt.Printf(Blue("Write quorum: %d \n", tenant.Status.WriteQuorum))
+	fmt.Printf(Blue("Drives online: %d \n", tenant.Status.DrivesOnline))
+	fmt.Printf(Blue("Drives offline: %d \n", tenant.Status.DrivesOffline))
+	fmt.Printf(Blue("Drives healing: %d \n", tenant.Status.DrivesHealing))
+	fmt.Printf(Blue("Health status: %s \n", tenant.Status.HealthStatus))
+	fmt.Printf(Blue("Capacity: %.1fGi \n", float32(tenant.Status.Usage.Capacity)/gb))
+	fmt.Printf(Blue("Raw capacity: %.1fGi \n", float32(tenant.Status.Usage.RawCapacity)/gb))
+	fmt.Printf(Blue("Raw usage: %.1fGi \n", float32(tenant.Status.Usage.RawUsage)/gb))
+}
+
+func (d *statusCmd) printJSONTenantStatus(tenant *v2.Tenant) error {
+	statusJSON, err := json.MarshalIndent(tenant.Status, "", "   ")
+	if err != nil {
+		return err
+	}
+	fmt.Printf(Green(fmt.Sprintf("JSON:\n\n%s\n\n", string(statusJSON))))
```

Comment:
```suggestion
	fmt.Printf(Green(fmt.Sprintf("JSON:\n\n%s\n\n", string(statusJSON))))
```

I think we can avoid adding `JSON:` this is not parsable by `jq` - instead we can simply have the actual json output just 

```
enc := json.NewEncoder(d.out)
enc.SetIndent("", " ")

if err := enc.Encode(tenant.Status); err != nil {
}
```